src/InsertionsClient.Core/Api/Providers/InsertionApi.cs

```diff
@@ -253,7 +265,7 @@ internal bool TryExtractManifestAssets(string manifestFile, out List<Asset> asse
             string json = File.ReadAllText(manifestFile);
             if (string.IsNullOrWhiteSpace(json))
             {
-                Trace.WriteLine($"Failed to read {InsertionConstants.ManifestFile} JSon content from file {manifestFile}.");
+                Trace.WriteLine($"Failed to read JSon content from file {manifestFile}.");
```

Comment:
nit: big S, or maybe even better, capitalize whole JSON

Reply:
Updated a couple of these cases in `Serializer.cs` as well.